plugins/modules/aci_cloud_subnet.py

```diff
@@ -260,9 +284,13 @@ def main():
     cloud_context_profile = module.params.get("cloud_context_profile")
     cidr = module.params.get("cidr")
     state = module.params.get("state")
-    availability_zone = module.params.get("availability_zone")
+    aws_availability_zone = module.params.get("aws_availability_zone")
+    azure_region = module.params.get("azure_region")
     child_configs = []
 
+    if aws_availability_zone and azure_region:
+        module.fail_json(msg="Configuring both an AWS availability zone and an Azure region is invalid.")
```

Comment:
Why not specify this as mutually_exclusive in Ansible module (line 269 above)? See documentation https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html or search our code base for mutually_exclusive.

Reply:
Done!